cards/src/main/resources/cards/custom/occultist/basic/spell_ancient_blood.json

```diff
@@ -1,16 +1,43 @@
 {
-  "name": "Ancient Blood",
-  "baseManaCost": 5,
+  "name": "Bloody Reconstruction",
+  "baseManaCost": 3,
   "type": "SPELL",
   "heroClass": "DARKGREEN",
   "rarity": "FREE",
-  "description": "Restore #12 Health.",
+  "description": "Restore [0] Health to your Champion. (Increases by 3 for each minion that died this turn.)",
+   "trigger": {
+    "eventTrigger": {
+      "class": "MinionDeathTrigger",
+      "targetPlayer": "BOTH"
+    },
+    "spell": {
+      "class": "ModifyAttributeSpell",

```

Comment:
This is currently doing increases by 3 each time a minion dies, not dies this turn. Use `DiedThisTurnValueProvider` instead of a `RESERVED_INTEGER` value provider.